WebUI/lims/views.py

```diff
@@ -255,15 +258,25 @@ def DeleteNotes(request, studyId, id):
 def createDatabase(request):
 
     # Prepare the connection string
-    username = ''    # TODO get the username from the form
-    password = ''    # TODO Get password from the form
+    username = request.POST['userName']    # TODO get the username from the form
+    password = request.POST['Password']    # TODO Get password from the form
     
     # Prepare the query
-    database = ''    # TODO Get database from form    
-    
+    database = request.POST['databaseName']    # TODO Get database from form
+    # Database name should be checked for compliance with SQL lexicon before the operation takes place
+    # Effectively, 1 to 31 characters, start with letter or underscore (_) may contain letters, numbers, or underscores after that.
+    result = regCheck("^[A-Za-z_][A-Za-z\d_]{0,31}$",database)
+    # If database name does not satisfy the requirements
+    if not result:
```

Comment:
See comment above, but just make the call to re as part of the if conditional.

Reply:
No problem